### PR TITLE
MAT-5895 Qi-Core supplemental data element single description field

### PR DESCRIPTION
**Pull request by sb-cecilialiu**
## MADiE PR

Jira Ticket: [MAT-5895](https://jira.cms.gov/browse/MAT-5895)
(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included.
* [x] No extemporaneous files are included (i.e Complied files or testing results).
* [x] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [x] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).
* [ ] All CDN/Web dependencies are hosted internally (i.e MADiE-Root Repo).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: src/components/editMeasure/populationCriteria/PopulationCriteriaHome.tsx [hunk 1]
```diff
@@ -112,10 +111,11 @@ export function PopulationCriteriaHome() {
...
       lazy(() => {
-        if (measure?.model === Model.QDM_5_6) {
-          return import("./supplementalData/qdm/QdmSupplementalElements");
-        } else {
-          return import("./supplementalData/qiCore/SupplementalElements");
+        if (
+          measure?.model === Model.QDM_5_6 ||
+          measure?.model === Model.QICORE
+        ) {
+          return import("./supplementalData/SupplementalData");
```
Comment (adongare) [inline src/components/editMeasure/populationCriteria/PopulationCriteriaHome.tsx]: does it make sense to remove the lazy loading now?

Comment (sb-cecilialiu) [reply src/components/editMeasure/populationCriteria/PopulationCriteriaHome.tsx]: @adongare I thought about that, then again, I wasn't sure if we support other types of measures we would use the same supplemental data form? Let me know if I should remove it thanks Ashok!

Comment (adongare) [reply src/components/editMeasure/populationCriteria/PopulationCriteriaHome.tsx]: I think we can remove it. we can add it later if we decide to support a new model.

Comment (sb-cecilialiu) [reply src/components/editMeasure/populationCriteria/PopulationCriteriaHome.tsx]: @adongare Ok thanks so much Ashok, will do.

Comment (sb-cecilialiu) [reply src/components/editMeasure/populationCriteria/PopulationCriteriaHome.tsx]: @adongare Removed.

**Review by adongare — DISMISSED** [review]

**Review by adongare — APPROVED** [review]

**Pull request merged**